### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by Sami-Ullah-Tufail**
### This branch has following features 
ddd
d3 data6
d3 data5
d3 data4
d3 data3
d3 data2
d3 data1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,291 +1,62 @@
...
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LICENSE -->
-
-## 📝 License <a name="license"></a>
+## License
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (abdulhamiid) [inline README.md]: The license link is broken, please fix it. Let's avoid broken links in our readme for a professional repository 😊🛠

image

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,92 @@
...
+-- List of all animals that are pokemon (their type is Pokemon).
+SELECT animals.name FROM animals RIGHT JOIN species ON animals.species_id =species.id  WHERE species.name = 'Pokemon'; 
+-- List all owners and their animals, remember to include those that don't own any animal.
+SELECT owners.full_name, animals.name FROM animals RIGHT JOIN owners ON owners.id = animals.owner_id;
+-- How many animals are there per species?
+SELECT species.name, count(species.name) FROM species LEFT JOIN animals ON species.id = animals.species_id GROUP BY species.name;
+-- List all Digimon owned by Jennifer Orwell.
+SELECT animals.name FROM  owners LEFT JOIN animals ON owners.id = animals.owner_id WHERE animals.species_id = 2 AND animals.owner_id = 2; 
+-- List all animals owned by Dean Winchester that haven't tried to escape.
+SELECT owners.full_name, animals.name, animals.escape_attempts FROM owners LEFT JOIN animals ON owners.id = animals.owner_id WHERE animals.escape_attempts = 0 AND owners.full_name = 'Dean Winchester';
```
Comment (abdulhamiid) [inline queries.sql:90]: ### Project requirements

image

As we were expected to add a screenshot of **each of our query results** to the pull request description, I could not find the screenshots of this query. ==> `List all animals owned by Dean Winchester that haven't tried to escape.`

Please add your screenshot. this helps to ensure that you run the queries with success. 🤓🤝

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Sami-Ullah-Tufail 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Use primary key & foreign key mechanism for joining tables.
- ✔️ Query multiple tables.
- ✔️ Prepare complex queries that answer analytical questions.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (Sami-Ullah-Tufail) [reply queries.sql:90]: i have added the screen shot it is the first one

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @Sami-Ullah-Tufail        , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  . 
Congratulations! 🎉  Keep up the Awesome work 🥂  Keep Rocking 🥇 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] Professional README file  
- [x] Database setup correctly 






 

 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

**Pull request merged**